loopy/transform/iname.py

```diff
@@ -1737,6 +1736,49 @@ def add_inames_to_insn(kernel, inames, insn_match):
 # }}}
 
 
+# {{{ remove_inames_from_insn
+
+@for_each_kernel
+def remove_inames_from_insn(kernel, inames, insn_match):
+    """
+    :arg inames: a frozenset of inames that will be added to the
+        instructions matched by *insn_match*, or a comma-separated
+        string that parses to such a tuple.
+    :arg insn_match: An instruction match as understood by
+        :func:`loopy.match.parse_match`.
+
+    :returns: a :class:`LoopKernel` with the *inames* removed from
+        the instructions matched by *insn_match*.
+
+    .. versionadded:: 2023.0
+    """
```

Comment:
Could you explain an example use case (in the docstring)?

Reply:
Done in 4727257eb214db4899f30ab02c757fafba30983d